examples/wearable/UIComponents/contents/etc/graphs/bar_graph.html

```diff
@@ -29,7 +29,7 @@ <h1>Bar Graph</h1>
 			 data-axis-x-type="order"
 			 data-axis-x-unit="s"></div>
         </div><!-- /content -->
-		<script src="../../lib/tau/libs/tau.graph.js" type="text/javascript"></script>
+		<script src="../../../lib/tau/libs/tau.graph.js" type="text/javascript"></script>
```

Comment:
Please check indents. Lokks like it should be aligned to script below since they are on the same parent (page).

Reply:
fixed